templates/senator/_layout.html.twig

```diff
@@ -27,18 +27,43 @@
                         </h1>
                     </div>
                     <nav class="manager-header__menu">
+                        {% set isSenator = is_granted('ROLE_SENATOR') %}
```

Comment:
I think we could avoid this check as we already are in `senator/_layout.html.twig`.
what about doing something like this: 

```twig
<ul>
  {% if is_granted('HAS_DELEGATED_ACCESS_...', 'senator') %}
    <li delegated_route>
  {% else %}
    <li senator_route>
  {% endif %}

  ...

</ul>
```

Reply:
ok done. I have still let it on the last tab, as "my team" should be displayed only to senator